### PR TITLE
Improve documentation of voltatge BT

### PR DESCRIPTION
**Pull request by Xevib**
### Description:

> Documentate the field voltatge

### Tasks:

- [x] Move menu instal·lacions
- [ ] Documentate the calculation of the BT voltatge

### Links:

* [Elements BT](http://builds.gisce.net/powerp-docs/powerp_imp_tensiones_lbt/ca/distri/lbt/#calcul-del-voltatge)
* [NOM DEL FITXER EN CASTELLÀ #TÍTOL-A-REVISAR](http://builds.gisce.net/powerp-docs/powerp_<branch_name>/es/<path/to/files>#<title-to-review>)

### CODE REVIEW

**Comment by giscegit** [comment]
Documentation build URL:
http://builds.gisce.net/powerp-docs/powerp_imp_tensiones_lbt/

Things to review:
- [ ] Typos in Catalan
- [ ] Typos in Spanish
- [ ] Links routes in Spanish
- [ ] Links work in Catalan
- [ ] Links work in Spanish
- [ ] Images in Spanish
- [ ] Page index shows all headers

**Comment by giscegit** [comment]
Documentation build URL:
http://builds.gisce.net/powerp-docs/powerp_imp_tensiones_lbt/

Things to review:
- [ ] Typos in Catalan
- [ ] Typos in Spanish
- [ ] Links routes in Spanish
- [ ] Links work in Catalan
- [ ] Links work in Spanish
- [ ] Images in Spanish
- [ ] Page index shows all headers

**Comment by giscegit** [comment]
Documentation build URL:
http://builds.gisce.net/powerp-docs/powerp_imp_tensiones_lbt/

Things to review:
- [ ] Typos in Catalan
- [ ] Typos in Spanish
- [ ] Links routes in Spanish
- [ ] Links work in Catalan
- [ ] Links work in Spanish
- [ ] Images in Spanish
- [ ] Page index shows all headers

**Comment by giscegit** [comment]
Documentation build URL:
http://builds.gisce.net/powerp-docs/powerp_imp_tensiones_lbt/

Things to review:
- [ ] Typos in Catalan
- [ ] Typos in Spanish
- [ ] Links routes in Spanish
- [ ] Links work in Catalan
- [ ] Links work in Spanish
- [ ] Images in Spanish
- [ ] Page index shows all headers

**Comment by giscegit** [comment]
Documentation build URL:
http://builds.gisce.net/powerp-docs/powerp_imp_tensiones_lbt/

Things to review:
- [ ] Typos in Catalan
- [ ] Typos in Spanish
- [ ] Links routes in Spanish
- [ ] Links work in Catalan
- [ ] Links work in Spanish
- [ ] Images in Spanish
- [ ] Page index shows all headers

File: docs/distri/lbt.md [hunk 1]
```diff
@@ -41,6 +41,20 @@ En l'apartat de **Dades Administratives** tenim els camps:
...
+
+- Si el cable del element de baixa tensio te assignat un nivell tensio 
+s'aplicara la "tensio a aplicar" del nivell de tensio assignat.
+  ![Voltatge LBT](_static/lbt/bt_nivell_tensio.png)
+  ![Nivell tensio cable](_static/lbt/bt_nivell_tensio.png)
+  ![Tensio a aplicar](_static/lbt/tensio_aplicar.png)
+- Altrement s'aplicara el que que marca cuadre de baixa tensio del centre transformador.
+
+!!!Note "Valor del voltatge"
+    El camp tensio no es un camp obligatori , pot estar en blanc, buid.
```
Comment (jaumef) [inline docs/distri/lbt.md]: buid → buit
tensio → tensió

File: docs/distri/lbt.md [hunk 2]
```diff
@@ -41,6 +41,20 @@ En l'apartat de **Dades Administratives** tenim els camps:
   * **Obres**: Amb quines obres relacionem aquest element.
   * **Expedients**: Amb quins expedients relacionem aquest element.
 
+## Calcul del voltatge
+
+El voltatge de un Element de Baixa tensió es calcula de la seguent manera:
+
+- Si el cable del element de baixa tensio te assignat un nivell tensio
```
Comment (jaumef) [inline docs/distri/lbt.md]: Crec que no fa falta fer una llista

**Comment by giscegit** [comment]
Documentation build URL:
http://builds.gisce.net/powerp-docs/powerp_imp_tensiones_lbt/

Things to review:
- [ ] Typos in Catalan
- [ ] Typos in Spanish
- [ ] Links routes in Spanish
- [ ] Links work in Catalan
- [ ] Links work in Spanish
- [ ] Images in Spanish
- [ ] Page index shows all headers

**Pull request merged**